README.md

```diff
@@ -54,27 +55,45 @@ If you want to open the OpenShift console at this stage, connect to the private
 data/status/sample/downloads/client.conf
 ```
 
-## 4. Configure OpenShift Data Foundation(ODF)
+It will take a while till you can view the resources created and in normal state. You can track the status of the resources created by reviewing the logs 
 
-This step is done by the automation. For background information visit [ui-docs/odf.md](./ui-docs/odf.md)
+Confirm the Satellite Location is created successfully
```

Comment:
I don't think this makes sense. If the satellite location failes to create, you'll see an error thrown by Ansible.